core/src/main/scala/org/apache/spark/serializer/KryoSerializer.scala

```diff
@@ -41,7 +41,7 @@ import org.apache.spark.internal.Logging
 import org.apache.spark.network.util.ByteUnit
 import org.apache.spark.scheduler.{CompressedMapStatus, HighlyCompressedMapStatus}
 import org.apache.spark.storage._
-import org.apache.spark.util.{BoundedPriorityQueue, SerializableConfiguration, SerializableJobConf, Utils}
+import org.apache.spark.util._
```

Comment:
Undo this unless there are new imports and it's getting really long, like 6 or 7 imports